packages/frontend/pages/api/twelvedata.ts

```diff
@@ -0,0 +1,20 @@
+import type { NextApiRequest, NextApiResponse } from 'next'
+
+const TWELVE_DATA_API = 'https://api.twelvedata.com'
+
+const handleRequest = async (req: NextApiRequest, res: NextApiResponse) => {
+  const { path } = req.query
+  const queryString = Object.keys(req.query).reduce((acc, key) => {
+    if (key !== 'path') acc += `${key}=${req.query[key]}&`
+    return acc
+  }, '')
+
+  if (!path) res.status(400).json({ status: 'error', message: 'Path parameter is missing' })
```

Comment:
Maybe we could add a return statement here, so execution stops here if the condition is true.

Reply:
oops yeah. Good catch!